web/src/engine/osk/src/keyboard-layout/gesturePreviewHost.ts

```diff
@@ -140,6 +153,10 @@ export class GesturePreviewHost extends EventEmitter<EventMap> {
     this.previewImgContainer.classList.add('flick-clear');
   }
 
+  public clearHint() {
+    this.hintLabel.classList.add('hint-clear');
+  }
```

Comment:
What does `clearHint()` / `hint-clear` do? It sounds like it is removing the hint but doing so by adding a class feels a bit strange. How is `hint-clear` removed again?

Also, it should be `kmw-hint-clear` for consistency and namespace protection

Reply:
If a key (somehow) supports both flicks and multitaps, `clearHint` will hide the multitap hint once a flick starts.  Granted, I don't think it would appear anyway, due to the order of events.

As a new `GesturePreview` instance is built for future gestures... `hint-clear` _isn't ever removed_.  There's no gesture life-cycle where undoing it makes sense... assuming that the preview instance is replaced when starting gestures.

Reply:
So ... then doesn't it make more sense to destroy it?